roles/openshift_docker_facts/tasks/main.yml

```diff
@@ -49,7 +49,7 @@
   when: not openshift.common.is_containerized | bool
 
 - set_fact:
-    l_common_version: "{{ openshift_version | default('0.0', True) | oo_image_tag_to_rpm_version }}"
+    l_common_version: "{{ openshift.docker.openshift_image_tag | default('0.0', True) | oo_image_tag_to_rpm_version }}"
```

Comment:
I'm pretty sure `openshift_version` is set in the defaults for the openshift_docker_facts role to be based on openshift.docker.openshift_image_tag.